libs/elements/cards/src/lib/elewa-info-card/elewa-info-card.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+import { InfoCard} from "libs/models/schema/ui/cards/info-card.interface"
```

Comment:
Update this to use aliased imports. (the file should be in a module that exports it)

Reply:
Hello, I hope you're well. I've made the change. I've updated the import statement to use an alias for the InfoCard interface. Please review the changes and let me know if it is what you had in mind.